### DISCUSSION TITLE
WBOT is spinning up

### DISCUSSION THREAD

**Discussion by wandy001** [vasani-arpit/WBOT#373] [Q&A]
why doesn't it run normally 5.8 and always plump in WBOT is spinning up.. and can't read the messages sent

**Comment by vasani-arpit** [comment]
Currently the problem is it can't reconnect to same session again. You can delete `chrome-data` and `.wwebjs` and try again. It will ask you for QR code and should work after that.

**Comment by wandy001** [comment]
Oke tq... 🙏🙏🙏🙏

Pada Sel, 16 Jul 2024 16.39, Arpit Vasani ***@***.***>
menulis:
